src/main/java/com/example/dada/repository/StudentRepository.java

```diff
@@ -0,0 +1,47 @@
+package com.example.dada.repository;
+
+import java.util.List;
+
+import org.jooq.DSLContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import static com.example.generated.tables.Student.STUDENT;
+import com.example.dada.dto.StudentDto;
+
+import lombok.RequiredArgsConstructor;
+@RequiredArgsConstructor
+@Repository
+public class StudentRepository {
+    private final DSLContext dslContext;
+  
```

Comment:
` var st = STUDENT.as("st");`
and use this `var` variable in your repository methods